velox/common/base/PeriodicStatsReporter.cpp

```diff
@@ -228,4 +232,13 @@ void PeriodicStatsReporter::reportCacheStats() {
   lastCacheStats_ = cacheStats;
 }
 
+void PeriodicStatsReporter::reportSpillStats() {
+  const auto spillMemoryStats = velox::memory::spillMemoryPool()->stats();
```

Comment:
Shall we just log delta spill stats? stats->toString() which shall include the peak and current memory usage? Thanks!

Reply:
This is memory pool stats which is snapshot. Sure we can have stats->toString()